src/actions/action_redirect.erl

```diff
@@ -19,9 +19,10 @@ render_action(#redirect{url=Url, login=Login}) ->
     wf:f("window.location=\"~ts\";", [wf:js_escape(RedirectUrl)]).
 
 -spec redirect(url()) -> html().
-redirect(Url) -> 
+redirect(Url) ->
     wf:wire(#redirect { url=Url }),
-    wf:f("<script>window.location=\"~ts\";</script>", [wf:js_escape(Url)]).
+    wf:f("<script>nonce=\"~s\" window.location=\"~ts\";</script>",
```

Comment:
This was the only issue I noticed in this PR.  The `nonce` should be inside the `<script>` so it was throwing JS errors.

Reply:
Oh, and I don't mean to say you should fix it.  I'm all over it. It's fixed here: https://github.com/nitrogen/nitrogen_core/commit/cd38d2d50bce48ca898dd0820c74c48e37cb0aaa

Reply:
Oh damn it.  Thanks for the catch.